src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,252 @@
+# Motion
+
+Support for Motion was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+Several [example Motion datasets](https://github.com/bids-standard/bids-examples)
+have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+## Motion recording data
+
+Template:
+
+└─ sub-<label>\
+└─ \[ses-<label>]\
+└─ motion\
+├─ sub-<label>\[\_ses-<label>]\_task-<label>\_tracksys-<label>\_motion.tsv
```

Comment:
I feel that analogous to `func/*_bold*` etc files, `task` should be optional here as well.  May be entire experiment/session is for a single task, or some composite task without clear name - so why to force it upon users unless it is needed to disambiguate two tasks?